argo/src/Argo.hs

```diff
@@ -699,11 +699,12 @@ serveHandlesNS hLog hIn hOut app =
 
 serveHTTP ::
```

Comment:
Would `serveHttp` better match the rest of the code?

Reply:
Done